downstream/modules/platform/ref-single-controller-hub-eda-with-managed-db.adoc

```diff
@@ -101,4 +104,61 @@ sso_keystore_password=''
 # automationedacontroller_ssl_cert=/path/to/automationeda.crt
 # automationedacontroller_ssl_key=/path/to/automationeda.key
 
+## Operational parameters
```

Comment:
@Alex-Izquierdo - I added the code from https://github.com/ansible/automation-platform-collection/blob/devel/roles/automationedacontroller/defaults/main.yml#L38 to the existing code sample in this section. Is that correct?

Reply:
The right person for that answer is @jamesmarshall24

IMO putting here the code of `defaults/main.yml` might be confusing for users because appear internal variables that the customer should not know about. 

We already have a list of the supported options in https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/red_hat_ansible_automation_platform_installation_guide/appendix-inventory-files-vars#event-driven-ansible-controller

Should not be there the place to add the new `automationedacontroller_max_running_activations` variable?

Reply:
defaults/main.yml documents variables for us as insiders to understand the value of configurable variables. Its not meant to be user facing without an editor in the middle to ensure the language is in alignment with their standards. I am not a docs writer and I wrote these descriptions to help, but I wouldn't consider my sentences the final copy of these descriptions. These shouldn't be added to the docs as presented in this PR.

Reply:
for this particular issue, we also shouldn't be copying and pasting the entire default variables file to the documentation. There was one variable added for this feature request, so what you want to use as a reference is just `automationedacontroller_max_running_activations`

Reply:
Thank you for your input, @jamesmarshall24 . I greatly appreciate it. Will make the corrections.